onyo/tests/test_usecases.py

```diff
@@ -0,0 +1,205 @@
+import subprocess
+import pytest
+from pathlib import Path
+
+from onyo.lib.onyo import OnyoRepo
+
+
+prefilled_assets = [["warehouse/monitor_dell_PH123.86JZho",
+                     "display: '22.0'\nfzj_inventory: '45FZ18'\n"],
```

Comment:
While this is not important for the test, I still want to mention that the fzj inventory numbers (as far as I know) are meant to be represented by the serial numbers in onyo.

Reply:
I don't think, that's a good idea. Discussed this with @aqw in context of #335 and he agreed to not be able to justify it anymore ;-)